From: Christoph Hellwig <hch@lst.de>
Date: Fri, 10 Jan 2025 08:37:32 +0100
Subject: [PATCH] loop: update commands in loop_set_status still referring to
 transfers
Git-commit: 4155adb01e7406653f6b01aaca916a59567cfbfa
Patch-mainline: v6.14-rc1
References: jsc#PED-9651

The concept of transfers is gone since commit 47e9624616c8 ("block:
remove support for cryptoloop and the xor transfer").

Signed-off-by: Christoph Hellwig <hch@lst.de>
Link: https://lore.kernel.org/r/20250110073750.1582447-3-hch@lst.de
Signed-off-by: Jens Axboe <axboe@kernel.dk>
Acked-by: Hannes Reinecke <hare@suse.com>
---
 drivers/block/loop.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/block/loop.c b/drivers/block/loop.c
index 6ea729cdce71..0c7dfc6eee12 100644
--- a/drivers/block/loop.c
+++ b/drivers/block/loop.c
@@ -1276,7 +1276,7 @@ loop_set_status(struct loop_device *lo, const struct loop_info64 *info)
 		invalidate_bdev(lo->lo_device);
 	}
 
-	/* I/O need to be drained during transfer transition */
+	/* I/O needs to be drained before changing lo_offset or lo_sizelimit */
 	blk_mq_freeze_queue(lo->lo_queue);
 
 	err = loop_set_status_from_info(lo, info);
@@ -1296,7 +1296,7 @@ loop_set_status(struct loop_device *lo, const struct loop_info64 *info)
 		loop_set_size(lo, new_size);
 	}
 
-	/* update dio if lo_offset or transfer is changed */
+	/* update the direct I/O flag if lo_offset changed */
 	__loop_update_dio(lo, lo->use_dio);
 
 out_unfreeze:
-- 
2.35.3

